.circleci/docker/build.sh

```diff
@@ -84,6 +84,15 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
+  pytorch-linux-xenial-cuda9.2-cudnn7-py3-gcc5.4)
+    CUDA_VERSION=9.2
+    CUDNN_VERSION=7
+    ANACONDA_PYTHON_VERSION=3.6
```

Comment:
We should probably just standardized on using the latest version of Python wherever possible.

Can you update this to Python 3.8?

Reply:
Sure, will do

Reply:
@seemethere 
I tried to build python 3.8 image, it seems to me that there is no simple way at this point to create such docker image. 
There are 2 things that blocking it:

1. Pytorch docker image install llvmlite 0.30.0 which does not support llvm9 under python 3.8, it need llvm-config which is not available in llvm9. 

2. Yes, I tried switching to llvmlite 0.31.0 (or 32), it works, the docker image can be built, but the gcc-5 version is upgraded to 5.5 instead of 5.4, this will affect all the pytorch docker image that using gcc-5, and currently, all the gcc-5 pytorch CI test assume it uses gcc 5.4,  so I think it would be better to make such change in a separate issue since this is not trivial.

I'd like to keep using python3.6 for this image, please advise.

Reply:
Sure let's stick to Python 3.6 here. Can you create a tracking issue to update these images to Python 3.8 with this information inside of the issue?

Reply:
sure. Created https://github.com/pytorch/pytorch/issues/37847
Will go back to 3.6 and request review agan.